src/routes/scheduleSync.js

```diff
@@ -16,7 +16,7 @@ module.exports = async (ctx) => {
       throw new Error(`Chill out and wait ${chillOutTimeSeconds}s`);
     }
     const test = ctx.query.test !== 'false';
-    const connectionOptions = readGwPubSubConfig();
+    const connectionOptions = config.gwPubSubConfig;
```

Comment:
Fix, howver needing testing waiting for: https://github.com/warp-contracts/warp-dre-node/issues/15#issuecomment-1382955269